### DISCUSSION TITLE
Svelte SSR/SSG in Go?

### DISCUSSION THREAD

**Discussion by jkdoshi** [pocketbase/pocketbase#534] [Q&A] [answered]
Hello PocketBase team, first of all huge thanks for creating PocketBase. It's awesome! I'm amazed how many features you have packed into such a small package. And not only is it a backend binary platform, it is also a framework.

Actually, I'm using it more as a framework (by compiling my own main.go that imports your module).

So at this point,  you've covered the backend API pretty well. Great! So I'm using SvelteKit's adapter-static to create my JAMStack apps with PB as the backend. Now, in order to avoid NodeJS as much as I can, I have to use fully static/prerendered frontend. But even there I have to use NodeJS at least during build time. It would be fantastic if PB took over the role of SvelteKit and did both SSR and SSG out of *.svelte files using Go instead of NodeJS. This framework called `livebud` is doing something like that [here](https://github.com/livebud/bud/tree/main/framework/view/ssr). But they don't have the full backend that PB has.

I know these are half-baked thoughts. And I'm sure my thinking will evolve as I spend more time on it, but I thought I'd put the idea in your head and get some feedback from you early as possible.

**Comment by ganigeorgiev** [comment, marked as answer]
At least for v1.0.0, I don't plan implementing SSR hydration in Go. It's a very complex topic and I'm not sure if I'll have the time to work on this.
